Header header
uint8[] audio_data
int32 bits_per_sample
int32 sample_rate
int32 number_of_channels
int32 number_of_frames
